wgpu-core/src/device/global.rs

```diff
@@ -241,7 +241,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             };
 
             let id = fid.assign(buffer, &mut token);
-            log::info!("Created buffer {:?} with {:?}", id, desc);
+            log::info!("Created Buffer {:?} with {:?}", id, desc);
```

Comment:
nitpick(non-blocking): This change and others like it, where the casing of a phrase otherwise present before is adjusted, seem insubstantial. I think an adjustment to match type names could be desirable, but it's not consistently implemented here. For example, `compute pipeline` and `command {buffer,encoder}` were missed.

Thoughts on dropping these changes, and optionally bringing them back with more polish would be better?